R/geom_roc.R

```diff
@@ -16,9 +16,9 @@ StatRoc <- ggproto("StatRoc", Stat,
                      } else {
                        data$group <- -1L
                      } 
-                   data
+                     data
                    },
-                   compute_group = function(data, scales, na.rm = TRUE){
+                   compute_group = function(data, scales, max_num_points = 1e3, na.rm = TRUE){
```

Comment:
Please move the max_num_points argument after na.rm, to be consistent with line 94